src/quo2/components/token_overview.cljs

```diff
@@ -0,0 +1,103 @@
+(ns quo2.components.token-overview
+  (:require
+   [quo2.foundations.colors :as colors]
+   [quo.react-native :as rn]
+   [clojure.string :as string]
+   [status-im.utils.currency :as currencies]
+   [status-im.ui.components.icons.icons :as icons]
+   [quo2.components.text :as text]))
+
+(def container-style {:display :flex :width "100%" :padding-left 20 :padding-right 20 :padding-top 12 :padding-bottom 12})
+
+(defn price-direction [price-change increase decrease neutral]
+  (if (> price-change 0) increase (if (< price-change 0) decrease neutral)))
+
+(defn price-color [direction]
+  (price-direction direction colors/success-50 colors/danger-50 colors/neutral-50))
+
+(defn divider [direction]
+  [rn/view {:style  {:height 10
+                     :margin-left 4
+                     :margin-right 4
+                     :width 1
+                     :background-color (price-direction direction colors/success-50-opa-40 colors/danger-50-opa-40 colors/divider-light)}}])
+
+(defn get-direction [percentage-change]
+  (if (= (js/parseInt percentage-change) 0) 0
+      (/ (js/parseInt percentage-change) (js/Math.abs  (js/parseInt percentage-change)))))
+
+(defn trim-minus [percentage-change]  (if (= (first percentage-change) "-") (string/join (rest percentage-change)) percentage-change))
+
+(defn token-price
+  "[token-price opts \"label\"]
+   opts
+   {
+    :currency :currency-key
+    :price :string
+    :percentage-change :string
+    }"
+  []
+  (fn
+    [{:keys [currency price  percentage-change] :or {currency :usd price "0.00" percentage-change "0.0"}}]
+    (let [direction (get-direction percentage-change)]
+      [rn/view {:style container-style}
+       [text/text  {:number-of-lines 1
+                    :size  :paragraph-2} "Token Price"]
```

Comment:
use labels for i18n

`(i18n/label :token-price)` and then you can add them to `translations/en.json` (only English is enough)

Reply:
Cool, sounds good :)